templates/vue-demo-store/app/components/checkout/cart/ProductItem.vue

```diff
@@ -85,7 +85,7 @@ const removeCartItem = async () => {
     class="mr-4 h-24 w-24 flex-shrink-0 overflow-hidden rounded-md border border-secondary-200"
   >
     <img
-      :src="getSmallestThumbnailUrl(cartItem.cover)"
+      :src="getSmallestThumbnailUrl(cartItem.cover?.media)"
```

Comment:
This can lead to empty `img` tags. We should check if `media` object exists in the item above